modules/@ergonode/core/src/models/arrayWrapper.js

```diff
@@ -360,3 +360,16 @@ export function firstElementInList(array) {
     const arrayLength = array.length;
     return arrayLength > 0 ? array[0] : null;
 }
+
+/**
+ * Async forEach method
+ * @function
+ * @param {Array} array
+ * @param {Function} callback
+ */
+export async function asyncForEach(array, callback) {
+    for (let index = 0; index < array.length; index += 1) {
+        // eslint-disable-next-line no-await-in-loop
+        await callback(array[index], index, array);
```

Comment:
It's synchronised forEach - might be confusing by name **asyncForEach**. To make it work in parallel I would suggest to make an array of requests and then resolving it all at once

Reply:
I disagree, it is an asynchronous loop and the name is not misleading. It is an asynchronous iteration over an array. This kind of solution is used when solving promises while iterating in a loop. 

I can of course use `Promie.all()`, what I wrote is not wrong just a different approach. 
I can write a second method with this solution, but it will be quite trivial.  

```javascript
await Promise.all(array.map(async (key) => {...}));
```

Reply:
I remove this method. method is not used